session_0/item.rb

```diff
@@ -23,14 +23,28 @@ def initialize(name: '', price: 0, quantity: 1, category_id: 4,
   # Returns a boolean value whether than item is discounted i.e. the
   # discount deadline has been crossed or not.
   def discounted?
-    raise NotImplementedError # TODO
+    if (Time.now > @discount_deadline) 
```

Comment:
- Use `discount_deadline` as we have define attr accessor for it.

```ruby
if condition
  return false
else
  return true
end
```

is same as:
```ruby
return !condition
```